cdc/log.cc

```diff
@@ -993,24 +996,36 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
                 // Note: further improvement here would be to coalesce the pre-image selects into one
                 // iff a batch contains several modifications to the same table. Otoh, batch is rare(?)
                 // so this is premature.
+                tracing::trace(tr_state, "CDC: Selecting preimage for {}", m.decorated_key());
                 f = trans.pre_image_select(qs.get_client_state(), db::consistency_level::LOCAL_QUORUM, m);
+            } else {
+                tracing::trace(tr_state, "CDC: Preimage not enabled for the table, so not selecting for {}", m.decorated_key());
             }
 
-            return f.then([trans = std::move(trans), &mutations, idx] (lw_shared_ptr<cql3::untyped_result_set> rs) mutable {
+            return f.then([trans = std::move(trans), &mutations, idx, tr_state = std::move(tr_state)] (lw_shared_ptr<cql3::untyped_result_set> rs) {
                 auto& m = mutations[idx];
                 auto& s = m.schema();
+                tracing::trace(tr_state, "CDC: Generating log mutations for {}", m.decorated_key());
+                int generated_count;
                 if (should_split(m, *s)) {
+                    tracing::trace(tr_state, "CDC: Splitting into several log mutations for {}", m.decorated_key());
```

Comment:
CDC: Splitting {} into several log mutations

Reply:
How about "CDC: Splitting {}"? Considering that trace messages are aligned to the right in cqlsh, keeping the key at the end of the message will make it easier to visually group messages for a particular key.

Reply:
ok.